src/main/java/org/junit/internal/runners/JUnit38ClassRunner.java

```diff
@@ -170,6 +174,13 @@ public void sort(Sorter sorter) {
         }
     }
 
+    public void order(GenericOrdering ordering) throws InvalidOrderingException {
```

Comment:
Because `Sorter` extends `Ordering`, but we want to make sure that anyone attempting to do a sort of a runner goes through `Ordering.apply(Object)`. Originally I didn't have  `Sorter` extend `Ordering`, and there the code was more complicated and had more duplication.